k8s/README.md

```diff
@@ -242,143 +263,74 @@ aws ecr create-repository \
-Notice this adds a tags for loading the image into the local docker registry created by `kind-with-registry.sh.` To load the image into the registry:
-```
-docker push "localhost:5000/zqd:latest"
-```
-This copies the image into a Docker registry that is accessed by your single-node Kubernetes cluster for Kind.
-
-## Redeploy with helm
-If you make a change to the helm templates, you generally will want to uninstall/reinstall zqd.
-```
-helm uninstall zqd-test-1
-helm install zqd-test-1 charts/zqd
-```
-To check if the AWS env vars are present in the deployment, these commands are helpful:
-```
-kubectl get deploy
-kubectl describe deploy zqd-test-1
-kubectl get deploy zqd-test-1 -oyaml
-```
-
-## Using zqd with zapi
-This is a walk-through of a local test before testing on our k8s cluster. This is outside the k8s cluster to get familiar with trouble-shooting.
-
-As a prerequisite, you must have an S3 bucket and directory available for zqd. In the S3 console, or at the command line, create a 'datauri' for use with zqd. This directory will hold metadata for the spaces you will create with zapi.
-
-You will need AWS credentials. On your local machine you can use `aws configure` and then you must set the environment variable, `AWS_SDK_LOAD_CONFIG=true`. Within Kubernetes, we will need to handle AWS credentials with secrets. More on that later.
+### Exposing the zqd endpoint
 
-Here is an example of creating a datauri, using an s3 bucket called `brim-scratch` in a directory called `mark` (change the s3 buckets for your setup.)
-```
-aws s3 ls # make sure the bucket exists!
-zqd listen -data s3://brim-scratch/mark/zqd-meta
-```
-zqd will stay running in that console, listening at `localhost:9867` by default.
-zqd will not create any s3 objects in zqd-meta until we issue a zapi command. Before using zapi, we use `zar import` in another console to copy sample data from our zq repo into s3:
-```
-zar import -R s3://brim-scratch/mark/sample-http-zng zq-sample-data/zng/http.zng.gz
-```
-This creates zng files in an s3 directory called `sample-http-zng` that we will use from zapi. To check what zar created:
-```
-aws s3 ls brim-scratch/mark --recursive
-```
-Now use zapi to make the zar data available to your running zqd instance. Note that zapi defaults to `localhost:9867` for connecting to zqd.
-```
-zapi new -k archivestore -d s3://brim-scratch/mark/sample-http-zng http-space-1
-```
-The `-d` parameter provides the same s3 dir that we used in the `-R` parameter of the zar command above. This command creates a new space for zqd called `http-space-1`. If we again list the s3 directories with `aws s3 ls brim-scratch/mark --recursive` we will see that zqd has now created a new object under its datauri, a directory name that is a base64 string prefixed with "sp_1g0".
+Work in progess...
 
-Now that you have a zqd running with a space, and you have made an archive from zar data, you can use zapi commands to query the sample data. Example:
+For now just use:
```

Comment:
FWIW, I was able to spin up a zqd pod, but I wasn't able to get any data back from it, using either Brim or zapi, after the port forwarding. I'll ping you off the PR with the logs, though they generally refer connection reset & timeouts, so perhaps there was some issue related to linkerd?

Reply:
I think this was cleared up on a subsequent run. Al had more than one occurrence of a non-fully functional EKS cluster being created.